src/Umbraco.Core/GuidUdi.cs

```diff
@@ -57,4 +58,19 @@ public GuidUdi EnsureClosed()
         EnsureNotRoot();
         return this;
     }
+
+    private static string CreateStringValue(ReadOnlySpan<char> entityType, Guid guid)
+    {
+        var startUdiLength = Constants.Conventions.Udi.StartUdi.Length;
+        var outputSize = entityType.Length + startUdiLength + 32 + 1; //Based on the format umb://entityType/guid (32 = Guid N format, 1 = / between entityType and guid)
+        Span<char> output = stackalloc char[outputSize];
+
+        //Add all the values of the format to the output
+        Constants.Conventions.Udi.StartUdi.CopyTo(output[..startUdiLength]);
+        entityType.CopyTo(output.Slice(6, entityType.Length));
```

Comment:
Just making sure the `6` should note be `startUidLength `😁

Reply:
Good point, it should be the startUidLength! Updated.